StepicUITests/Sources/StepicUITests.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  StepicUITests.swift
+//  StepicUITests
+//
+//  Created by AKholin on 11.03.2021.
+//  Copyright © 2021 Alex Karpov. All rights reserved.
+//
+
+import XCTest
+
+class StepicUITests: XCTestCase {
+    
+
+    override func setUpWithError() throws {
```

Comment:
Настройка состояния не может бросить ошибку, поэтому можно использовать обычный метод `setUp()`.
```suggestion
    override func setUp() {
        super.setUp()
```